### PR TITLE
[test] Clang diagnostic reports in skip tests

### PR DESCRIPTION
**Pull request by bruntib**
Skip test is checking whether some reports are really not
stored because they were skipped during analysis with --skip flag.
However, since clang-tidy is not run, there will be no reports from
"clang-diagnostic" checker. These are handled separately here,
otherwise the test would believe they're missing because of --skip
which is not the case.

### CODE REVIEW

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- web/tests/functional/skip/test_skip.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=6351612&bid=20624915)

**Review by gyorb — APPROVED** [review]

File: web/tests/functional/skip/test_skip.py [hunk 1]
```diff
@@ -93,7 +101,8 @@ def test_skip(self):
 
         if missing_results:
             for bug in missing_results:
-                self.assertIn(bug['file'], skipped_files)
+                if not bug['checker'].startswith('clang-diagnostic'):
```
Comment (csordasmarton) [inline web/tests/functional/skip/test_skip.py:104]:
```suggestion
                if not bug['checker'].startswith('clang-diagnostic-'):
```

**Review by csordasmarton — CHANGES_REQUESTED** [review]
Also please update the commit with a tag: `[test] ...`

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- web/tests/functional/skip/test_skip.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=6351612&bid=20624915)

**Review by csordasmarton — APPROVED** [review]
LGTM!

**Pull request merged**